packages/erc3k/contracts/IERC3000Executor.sol

```diff
@@ -8,7 +8,12 @@ pragma experimental ABIEncoderV2;
 import "./ERC3000Data.sol";
 
 abstract contract IERC3000Executor {
+    // Note: how is this expected to treat failing actions? especially when having multiple actions but one of them fails
```

Comment:
This is an interesting comment.

We could add a 'allowFailures' bitmap argument in which you can specify which action indexes are allowed to fail and which not